e2e/tests/functional/plugins/notebook/tags.e2e.spec.js

```diff
@@ -231,4 +208,25 @@ test.describe('Tagging in Notebooks @addInit', () => {
             await expect(page.locator(entryLocator)).toContainText("Driving");
         }
     });
+    test('Can cancel adding a tag', async ({ page }) => {
+        await createNotebookAndEntry(page);
+
+        // Click on Annotations tab
+        await page.locator('text=Annotations').click();
```

Comment:
We should update this selector to be more deterministic. Ideally we could select by role `tab`, but we could also narrow down the selector by using classes in addition to the label.

Something like this should work:

```js
// Click on Annotations tab
await page.locator('.c-inspector__tab', { hasText: "Annotations" }).click();
```